examples/zig/tokamak/hello-world/src/main.zig

```diff
@@ -11,6 +11,7 @@ const App = struct {
 };
 
 pub fn main() !void {
+    std.debug.print("Running on port 8080\n", .{});
```

Comment:
I had to google the default port for tokamak so figured i'd just print it to save people the trouble in the future ...